### ISSUE TITLE
Make master the default branch and remove develop

### ISSUE THREAD

**Issue by szymon-jez** [socialpaymentsbv/ex_phone_number#30]
I see no use-case for having those 2 branches at once.
Not having `master` as the default branch can be confusing. See https://github.com/socialpaymentsbv/ex_phone_number/pull/27#issuecomment-465485566.

**Comment by jc00ke** [comment]
Or `main`?

**Comment by josemrb** [comment]
I'm using a light version of the Gitflow workflow, the `master` branch stores the published code, and the `develop` branch serves as an integration branch for the feature branches.

Once the new version in `develop` is ready, it gets merged into `master` and tagged with a version number.

> See [#27 (comment)](https://github.com/socialpaymentsbv/ex_phone_number/pull/27#issuecomment-465485566).

The problem was that the `master` branch was not updated but I have already fixed it.

**Comment by szymon-jez** [comment]
@josemrb Then if this is the case let's document this in the README.
